keyboards/keebio/levinson/rev2/config.h

```diff
@@ -43,6 +43,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* serial.c configuration for split keyboard */
 #define SOFT_SERIAL_PIN D0
+#define NO_RIGHT_HALF_EXT_I2C_PULLUPS
```

Comment:
So my levinson has pullups on both halfs to avoid this issue, how does this setting interact with that config?

Reply:
From reading around, it looks like the internal pullup resistance is around 20k-40kΩ. Assuming worst case of 20kΩ, the total parallel resistance of both halves combined would go from 2.35kΩ (for two 4.7kΩ resistors) to 2.1kΩ (2 4.7kΩ w/1 20kΩ), which is still in a reasonable range for I2C.